src/config.ts

```diff
@@ -96,6 +96,8 @@ export default {
       'pop',
       'fellow',
       'fellowship',
+      'basically',
+      'long time no see',
```

Comment:
Rule isn't named correctly - it should be `long-time-no-see`.

see documentation here:
https://github.com/retextjs/retext-profanities/blob/main/rules.md
https://github.com/retextjs/retext-equality/blob/main/rules.md

Reply:
should i make another PR?

Reply:
No, Just adjust this one. 👍